Assets/BossRoom/Scripts/Game/Client/UI/RoomNameBox.cs

```diff
@@ -4,6 +4,7 @@
 using Unity.Multiplayer.Samples.BossRoom.Shared.Infrastructure;
 using Unity.Multiplayer.Samples.BossRoom.Shared.Net.UnityServices.Lobbies;
 using UnityEngine.UI;
+using VContainer;
```

Comment:
I'll stop creating comments for all of these, I'd make sure to review the PR for all of these

Reply:
@fernando-cortez I'm actually surprised CI didn't catch this?

Reply:
I've noticed that too. I believe the formatting doesn't actually check for unnecessary directives, though I think it can. I will look into that.

Reply:
This is not an unnecessary directive.